nilearn/plotting/glass_brain.py

```diff
@@ -99,7 +99,7 @@ def _get_json_and_transform(direction):
         "r": [0.38, 0, 0, 0.38, -108, -70],
     }
 
-    dirname = os.path.dirname(os.path.abspath(__file__))
+    dirname = os.path.dirname(Path.resolve(__file__))
     dirname = os.path.join(dirname, "glass_brain_files")
```

Comment:
I think we can even do this.

```suggestion
    dirname = Path(__file__).resolve().parent / "glass_brain_files"
```